lms/djangoapps/edraak_university/models.py

```diff
@@ -1,20 +1,19 @@
 from django.db import models
 from django.utils import timezone
 from django.contrib.auth.models import User
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext_lazy as _, get_language
 
 from student.models import UserProfile
 
 from xmodule_django.models import CourseKeyField
 
-
 class UniversityID(models.Model):
     user = models.ForeignKey(User)
     course_key = CourseKeyField(max_length=255, db_index=True)
     university_id = models.CharField(verbose_name=_('Student University ID'), max_length=100)
     section_number = models.CharField(verbose_name=_('Section Number'), max_length=10)
     date_created = models.DateTimeField(default=timezone.now)
-
+    terms_and_conditions_read = models.BooleanField(default=False)
```

Comment:
- [ ] Not needed, just remove it and all the users should know that submitting is agreeing by default.